script/1_0_2current_en_script/Ciel/Day 3/03_01_CIEL03_2_3.txt

```diff
@@ -113,12 +113,12 @@ Really, you shouldn't be following strangers all the time. ...If the culprit is
 "...No, um, Senpai? I'm not sure if that's a good comparison."
 <Page36>
 Don't say such ominous things.
-I have chronic anemia in the first place so I'm probably not a target for vampires.
+I have chronic anemia in the first place so I'm probably not a good target for vampires.

```

Comment:
I have chronic anemia, so I'm probably not a good target for vampires in the first place.